### PR TITLE
Fixes #8372: Several systemd warnings in install on RHEL/CentOS 7  bugs in original resolution and missing changes

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/8372

### CODE REVIEW

File: rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec [hunk 1]
```diff
@@ -173,8 +173,14 @@ echo "INFO: Launching script to check if a migration is needed"
...
 echo "INFO: End of migration script"
 
-echo "Restarting syslogd ..."
-service %{syslogservicename} restart
+echo -n "INFO: Restarting syslogd ..."
+%if 0%{?rhel} < 7
+service %{syslogservicename} restart > /dev/null && echo " Done"
+%endif
+%if 0%{?rhel} >= 7
+/bin/systemctl restart  rsyslog.service && echo " Done"
```
Comment (peckpeck) [inline rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec]: You could use %{syslogservicename}.service instead

File: rudder-reports/SPECS/rudder-reports.spec [hunk 2]
```diff
@@ -117,14 +117,26 @@ else
 fi
 
 # Check if PostgreSQL is started
-service ${POSTGRESQL_SERVICE_NAME} status > /dev/null
+%if 0%{?rhel} < 7
+CHECK_COMMAND="service ${POSTGRESQL_SERVICE_NAME} status"
```
Comment (peckpeck) [inline rudder-reports/SPECS/rudder-reports.spec]: Why do you do differently here and use a variable instead of a direct call ?

**Comment by jooooooon** [comment]
Commit modified

File: rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec (hunk 1 above)
Comment (jooooooon) [inline rudder-inventory-endpoint/SPECS/rudder-inventory-endpoint.spec]: Good catch, fixing

**Comment by jooooooon** [comment]
Commit modified

**Comment by peckpeck** [comment]
Good, merging

**Pull request merged**